### PR TITLE
Feature red logo

### PR DESCRIPTION
**Pull request by curtisahuang**
## Description

Matching color theme and updating logo to red version

## Closes issue(s)

If there any issue that is fixed with this PR, please attach it/them here...

## Screenshots

with red niwa

## Changes include

- [X] UI/UX or color-theme
- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

## Checklist

- [ ] I have tested this code
- [ ] Add here the checks for this PR. If any please delete these checks

### CODE REVIEW

**Review by conalli — COMMENTED** [review]
Would it be possible to change the black to #172F4A to match the rest of the theme?

**Comment by curtisahuang** [comment]
I'm afraid that I just blue myself

blueversion

File: src/pages/Home/home.css [hunk 1]
```diff
@@ -19,9 +19,11 @@
 .ni-wa {
   font-size: 6rem;
   line-height: 1rem;
+  color: #172f4a;
 }
```
Comment (daiHash) [inline src/pages/Home/home.css:23]: Its Ok to do in a different PR 
we can define css variables for the values of styles that we're going to reuse throughout the app, so editing would be easier

**Review by daiHash — APPROVED** [review]
👍

**Review by conalli — APPROVED** [review]
